syncopy/tests/test_preproc.py

```diff
@@ -0,0 +1,235 @@
+# -*- coding: utf-8 -*-
+#
+# Test preprocessing 
+#
+
+# 3rd party imports
+import psutil
+import pytest
+import inspect
+import numpy as np
+import matplotlib.pyplot as ppl
+
+# Local imports
+from syncopy import __acme__
+if __acme__:
+    import dask.distributed as dd
+
+from syncopy import preprocessing as pp
```

Comment:
I think using a somewhat more telling import shortcut like `ppfunc` or `preprocfunc` might make the tests a little easier to read

Reply:
I had the exact opposite impression, as code lines become very long and hence less readable.. for the connectivity tests it is the same actually (`import connectivityanalysis as ca`). But I changed it to `ppfunc` :)

Reply:
Awesome, thanks!